src/test/java/core/basesyntax/service/impl/FileReaderImplTest.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Before;
+import org.junit.Test;
+
+public class FileReaderImplTest {
+    private static final String SOURCE_FILE = "src/main/java/resources/filetest";
+    private final FileReaderImpl fileService = new FileReaderImpl();
```

Comment:
```suggestion
    private final FileReaderImpl fileReader = new FileReaderImpl();
```

it's more informative